packages/dashboard/components/consent/utils/consentTransactionUtils.ts

```diff
@@ -193,6 +202,8 @@ export const addPermittedActions = async (
         pkpTokenId: agentPKPTokenId,
         authMethodScopes: [AUTH_METHOD_SCOPE.SignAnything],
       });
+      console.log(`Added permission for ${ipfsCid} - Transaction hash: ${tx.transactionHash}`);
+
     } catch (error) {
       console.error(
         `Error adding permitted action for IPFS CID ${ipfsCid}:`,
```

Comment:
See my comment above -- if this code fails, we must set an error state of some kind and reflect the failure to the user, otherwise we are just sending the user off to an app that isn't actually functional, and where the app developer has no way to actually resolve the issue other than sending the user right back here.